src/main.c

```diff
@@ -454,6 +466,33 @@ main(int argc, char* argv[])
             printf("\rDialog count: %d\n", sip_calls_count_unrotated());
     }
 
+
+    if (siplify_outfile)
+    {
+        no_interface = 1;
```

Comment:
This is not probably required, because the only way to enter this if is through the above case, who already set this value to no_interface.